database/seed/postgres/seedRandomData.js

```diff
@@ -7,10 +7,10 @@ const seed = async () => {
     let products = 6000000;
     let reviewId = 0;
   
-    await client.query('DELETE FROM review_summaries WHERE summary_id >= 0');
+    await client.query('DELETE FROM review_summaries WHERE product_id >= 0');
     await client.query('DELETE FROM reviews WHERE review_id >= 0');
     await client.query('BEGIN');
-    await client.query('PREPARE insert_summary (int, int, int, int, int, int, int, int) AS INSERT INTO review_summaries VALUES ($1, $2, $3, $4, $5, $6, $7, $8) RETURNING product_id, rating_1, rating_2, rating_3, rating_4, rating_5');
+    await client.query('PREPARE insert_summary (int, int, int, int, int, int, int) AS INSERT INTO review_summaries VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING product_id, rating_1, rating_2, rating_3, rating_4, rating_5');
     await client.query('PREPARE insert_review (int, int, char(30), char(30), char(30), int) AS INSERT INTO reviews VALUES($1, $2, $3, $4, $5, $6)');
```

Comment:
`char(30)` wouldn't be enough to contain review texts.

Reply:
I'll push a fix in a different pull request